tests/unit/test_runtime_build.py

```diff
@@ -90,7 +90,7 @@ def test_prep_build_folder(temp_dir):
         )
 
     # make sure that the code was copied
-    shutil_mock.copytree.assert_called_once()
+    assert shutil_mock.copytree.call_count == 2
```

Comment:
Could you modify the comment to explain this?

Reply:
done!